vm/src/stdlib/itertools.rs

```diff
@@ -569,10 +568,7 @@ mod decl {
             }
 
             // might be StopIteration or anything else, which is propagated upwards
-            let obj = match zelf.iterable.next(vm)? {
-                PyIterReturn::Return(obj) => obj,
-                PyIterReturn::StopIteration(v) => return Ok(PyIterReturn::StopIteration(v)),
-            };
+            let obj = raise_stop!(zelf.iterable.next(vm)?);
```

Comment:
uh, my fault. now returning a value from `raise_stop!`  looks awkward.

```suggestion
            let obj = raise_if_stop!(zelf.iterable.next(vm)?);
```
I will add `if` in the middle